test/gurka/test_node_access.cc

```diff
@@ -0,0 +1,73 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+
+TEST(Standalone, NodeAccess) {
```

Comment:
IS this test related to the sump_buster change?

Reply:
@mandeepsandhu no. This is an access bug that I found during the testing of sump_buster.